source/documentation/data-docs/curated-databases-docs/databases-for-analysis-and-apps.md

```diff
@@ -0,0 +1,56 @@
+
+- If you need to do joins across datasets it is most likely that Athena is going to be much faster than Python/R unless your datasets are very small. So if you are doing some analysis with many joins use Athena.
+
+If in doubt about best practices we have slack channels for R, python, SQL, etc where you can ask.
+
+## Guidance on using databases / data for deployed apps
+
+This section of the guidance covers best practice for using data on the platform for deployed apps. If you want to know how to set up an app bucket go to the [deploy an R shiny app section](#deploying-an-r-shiny-app) of the guidance.
+
+### What to consider when giving an app access to data
+
+This is because apps are typically accessed by users without Analytical Platform accounts. To add additional security for our data, we like to restrict how much data apps can access, in order to prevent unwanted data leaks in the unlikely event of an app being compromised. In order to describe how this affects things when we give apps access to databases, we need to briefly touch on 'roles' within the Analytical Platform.
+
+When you access things on the Analytical Platform, you do so by taking on your 'role', which lists the set of things that you are permitted to do. This is how you interact with RStudio, Jupyter, Airflow, S3 buckets, etc. Since apps also need permissions to access things, they also have roles. When app users are interacting with the app, it will be carrying out tasks using its own role. In order to minimise the damage that an app could do in the unlikely event of it being compromised, we provide apps with much more restricted roles than those given to users. What this means is:
+
+- Apps cannot access (read/write) S3 buckets created by the AP users (instead they have their own 'app buckets' prefixed with `alpha-app-`)
+- Apps cannot access databases created for the AP users (these are all of the "curated" databases created by the Data Engineering Team)
+- Users can access read/write to app buckets (permissions to do this are given via the Control Panel).
+
+So if you create an app that needs specific data from our databases then you need to consider the following practices:
+
+#### Data Minimisation
+
+When writing data to your app bucket you should only ever give the app the minimal data that it needs. For example, even though your analysis might require personal or low level data, your app may well be presenting this data at an aggregated level, in which case you should put just the aggregated data into the app bucket. You should never put data in the app bucket that you do not want users to see (even if the code behind the app itself has no method to expose that data to the user via the app).
```

Comment:
Second sentence is cleaner if it's something like
"For example, if your app presents analytical results at an aggregated level, you should put just the aggregated data into the app bucket, even if the underlying analysis that creates these results requires personal or low level data."